docs/mkdocs/docs/api/basic_json/start_pos.md

```diff
@@ -0,0 +1,67 @@
+# <small>nlohmann::basic_json::</small>start_pos
+
+```cpp
+#if JSON_DIAGNOSTIC_POSITIONS
+constexpr std::size_t start_pos() const noexcept;
+#endif
+```
+
+Returns the position of the first character of the JSON string the value was parsed from.
```

Comment:
parsed from, if the value was parsed from a string.  If this value was constructed in some other way, it will return `std::string::npos`.

Reply:
Done in 37bb4d9c5d504ff0e8a9cd00cb33c84009ebcf95